core/src/test/java/com/linecorp/armeria/client/endpoint/dns/DnsNameEncoder.java

```diff
@@ -18,9 +18,9 @@
 import io.netty.buffer.ByteBuf;
 import io.netty.handler.codec.dns.DefaultDnsRecordEncoder;
 
-final class DnsNameEncoder {
+public final class DnsNameEncoder {
 
-    static void encodeName(String name, ByteBuf out) {
+    public static void encodeName(String name, ByteBuf out) {
```

Comment:
If we're going to make this class public, we have to hide it from the public API. Could you move this class to `internal.client`? We could also merge it into `internal.client.DnsUtil`.